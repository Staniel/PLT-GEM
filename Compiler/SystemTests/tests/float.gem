void main(String[] args) {
	print 12.43+0.12;
}