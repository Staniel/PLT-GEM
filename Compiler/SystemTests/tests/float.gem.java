import java.util.*;public class Main {
public static Scanner scanner = new Scanner(System.in);public static void main(String[] args){
System.out.println(12.43 + 0.12);
}
}

