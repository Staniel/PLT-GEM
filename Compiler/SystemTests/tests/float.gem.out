12.549999999999999
